module/x/peggy/keeper/keeper.go

```diff
@@ -321,65 +380,12 @@ func (k Keeper) SetOrchestratorValidator(ctx sdk.Context, val sdk.ValAddress, or
 // GetOrchestratorValidator returns the validator key associated with an orchestrator key
 func (k Keeper) GetOrchestratorValidator(ctx sdk.Context, orch sdk.AccAddress) sdk.ValAddress {
```

Comment:
Adding the other direction (i.e. `GetValidatorFromOrchestrator`) could be helpful too. I've found this nice in the Oracle client progamming.